src/Tribe/Aggregator/Cron.php

```diff
@@ -388,6 +390,81 @@ public function verify_child_record_creation() {
 		}
 	}
 
+	/**
+	 * Start the processing of the scheduled imports created with batch pushing the cron job would select and start
+	 * the beginning of the batch delivery.
+	 *
+	 * @since TBD
+	 */
+	public function start_batch_pushing_records() {
+		if ( ! tribe( 'events-aggregator.main' )->is_service_active() ) {
+			return;
+		}
+
+		$records = Tribe__Events__Aggregator__Records::instance();
+
+		$query = $records->query(
+			[
+				'post_status'    => Tribe__Events__Aggregator__Records::$status->pending,
+				'posts_per_page' => 250,
```

Comment:
This could be extremely painful depending on what we get back. There are 3 meta values here.

Reply:
By painful do you mean slow?

Actually this one is a bit more conservative that the regular cron for EA client.

https://github.com/moderntribe/the-events-calendar/blob/674025ca52e6cab552b78026186f73bd027dbfde/src/Tribe/Aggregator/Cron.php#L483-L484